### PR TITLE
Dropped isomorphic-fetch in examples in favor of cross-fetch (React Native compatible).

### PR DESCRIPTION
**Pull request by lquixada**

### CODE REVIEW

**Comment by timneutkens** [comment]
I'd rather change to isomorphic-unfetch for everything. Next is using `unfetch` itself and we don't support react-native. https://github.com/developit/unfetch/tree/master/packages/isomorphic-unfetch

**Comment by albinekb** [comment]
Really? 🤔

File: examples/data-fetch/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "start": "next start"
   },
   "dependencies": {
-    "isomorphic-fetch": "^2.2.1",
+    "cross-fetch": "^0.0.8",
```
Comment (albinekb) [inline examples/data-fetch/package.json]: Why change out a well tested lib to a lib that is not even 1.0.0 and does exactly the same thing?

**Comment by timneutkens** [comment]
Lets close this 👍 As said I'd much rather change to isomorphic-unfetch. It's the smallest solution around
bundlephobia.com/result?p=cross-fetch
bundlephobia.com/result?p=isomorphic-fetch
bundlephobia.com/result?p=isomorphic-unfetch

**Pull request closed without merging**